parsl/providers/cobalt/cobalt.py

```diff
@@ -48,8 +48,10 @@ class CobaltProvider(ClusterProvider, RepresentationMixin):
         Account that the job will be charged against.
     queue : str
         Torque queue to request blocks from.
-    overrides : str
-        String to append to the Torque submit script on the scheduler.
+    scheduler_options : str
+        String to prepend to the submit script to the scheduler.
+    worker_init : str
+        Command to be run before running a task, such as 'module load Anaconda; source activate env'.
```

Comment:
We have two different docstrings, but really these two args are going to the same place and doing the same thing (being prepended to the submit script to the scheduler). I am sure there was a good reason for splitting it up like this, but I can't recall now what it was. I've reached my quota of opinining on this issue, so I recuse myself from promoting any particular course of action, while noting that I find this choice a bit confusing. In other words.... ignore this paragraph, sorry for the noise.

Also, "Command to be run before running a task" might make people think this executed once per task, while it is really once per batch script (even though it is technically before running a task). Maybe "Command to be run before starting a worker".